watchtower-plugin/src/main.rs

```diff
@@ -586,41 +582,46 @@ async fn main() -> Result<(), Error> {
         )
         .hook("commitment_revocation", on_commitment_revocation);
 
-    if let Some(plugin) = builder.start().await? {
-        // FIXME: This is a workaround. Ideally, `cln_plugin::options::Value` will implement `as_u64` so we can simply call and unwrap
-        // given that we are certain the option exists.
-        state_clone.lock().unwrap().proxy =
-            if let Value::String(x) = plugin.option("watchtower-proxy").unwrap() {
-                if !x.is_empty() {
-                    Some(x)
-                } else {
-                    None
-                }
+    // We're unwrapping here given it does not seem we actually have anything to check at the moment.
+    // Change this so the plugin can be disabled soon if this happens not to be the case.
+    let midstate = builder.configure().await?.unwrap();
```

Comment:
I am not able to manually try this out ATM. Does this unwrap log anything to the terminal when `lightningd` is called with `--help`. Or does it mess up with the exit code of the `lightningd` process?

I think this could better be handled with:
```rust
let midstate = if let Some(midstate) = builder.configure().await? {
	midstate
} else {
	return Ok(())
};
```

If the `unwrap` is benign though, no need to do this.

Reply:
Yep, this is right. I'll patch it.